### PR TITLE
Added unit prefixes to documentation(Issue #5511) 

### PR DESCRIPTION
**Pull request by vibhorgupta96**
Added unit prefixes
https://github.com/astropy/astropy/issues/5511

### CODE REVIEW

**Comment by pllim** [comment]
Note: Note: There is an alternate PR submitted by someone else as #5648

**Comment by vibhorgupta96** [comment]
I saw that the issue was open for about 28 days and there was no PR so I did it.

**Comment by pllim** [comment]
@vibhorgupta96 , no worries. In fact, the other PR was submitted *after* yours. Multiple solutions to the same problem do occur from time to time, especially in an open-source project like Astropy. Package maintainer(s) will review both and choose one. Thank you for contributing!

**Comment by pllim** [comment]
You need to fix these Sphinx errors:
```
/home/travis/build/astropy/astropy/docs/units/standard_units.rst:87: WARNING: Title underline too short.

Binary Prefixes

^^^^^^^^^^^^^^

/home/travis/build/astropy/astropy/docs/units/standard_units.rst:87: WARNING: Title underline too short.

Binary Prefixes

^^^^^^^^^^^^^^
```

**Comment by vibhorgupta96** [comment]
@pllim I have fixed the sphinx errors.

**Comment by vibhorgupta96** [comment]
@pllim I am quite free right now and wouldn't mind working on any modifications you suggest.

File: docs/units/standard_units.rst [hunk 1]
```diff
@@ -48,6 +48,56 @@ all the existing predefined units of a given type::
...
+     y     |  yocto     | 1e-24
+
+
+Binary Prefixes
+^^^^^^^^^^^^^^^
+
+A binary prefix is a unit prefix for multiples of units in data processing, data transmission, and digital information, notably the bit and the byte, to indicate multiplication by a power of 2. The following are supported::
+
+
+     Symbol | Unit name | Value
```
Comment (pllim) [inline docs/units/standard_units.rst]: Same comment as above.

File: docs/units/standard_units.rst [hunk 2]
```diff
@@ -48,6 +48,56 @@ all the existing predefined units of a given type::
...
+----------------------
+
+In addition to these units, `astropy.units` includes the concept of unit prefixes. A unit prefix is a specifier or mnemonic that is prepended to units of measurement to indicate multiples or fractions of the units. Units of various sizes are commonly formed by the use of such prefixes.
+
+SI Prefixes
+^^^^^^^^^^^
+The prefixes of the metric system (commonly known as SI prefixes) precede a basic unit of measure to indicate a decadic multiple (base 10) and fraction of a unit. The following are supported::
+
+
+    Symbol | Unit name  | Value
```
Comment (pllim) [inline docs/units/standard_units.rst]: Instead of `::` block quote, it would be nicer to have a properly formatted table.

**Comment by mhvk** [comment]
@vibhorgupta96 - it is a bit unfortunate that there were two PRs, both of which were essentially fine. I went with #5654, since that was already in table format, but would still want to thank you very much for your PR!

**Pull request closed without merging**